src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -262,34 +262,35 @@ TR1   TRIG  n/a   n/a         n/a         good    n/a
-| ECG      | ElectroCardioGram (heart)                                         |
-| EMG      | ElectroMyoGram (muscle)                                           |
-| TRIG     | System Triggers                                                   |
-| AUDIO    | Audio signal                                                      |
-| PD       | Photodiode                                                        |
-| EYEGAZE  | Eye Tracker gaze                                                  |
-| PUPIL    | Eye Tracker pupil diameter                                        |
-| MISC     | Miscellaneous                                                     |
-| SYSCLOCK | System time showing elapsed time since trial started              |
-| ADC      | Analog to Digital input                                           |
-| DAC      | Digital to Analog output                                          |
-| REF      | Reference channel                                                 |
-| OTHER    | Any other type of channel                                         |
+| Keyword  | Description                                                            |
+| :------- | :--------------------------------------------------------------------- |
+| EEG      | Electrode channel from electroencephalogram                            |
+| ECOG     | Electrode channel from electrocorticogram (intracranial)               |
+| SEEG     | Electrode channel from stereo-electroencephalogram (intracranial)      |
+| DBS      | Electrode channel from deep brain stimulation electrode (intracranial) |
+| VEOG     | Vertical EOG (electrooculogram)                                        |
+| HEOG     | Horizontal EOG                                                         |
+| EOG      | Generic EOG channel if HEOG or VEOG information not available          |
+| ECG      | ElectroCardioGram (heart)                                              |
+| EMG      | ElectroMyoGram (muscle)                                                |
+| TRIG     | System Triggers                                                        |
+| AUDIO    | Audio signal                                                           |
+| PD       | Photodiode                                                             |
+| EYEGAZE  | Eye Tracker gaze                                                       |
+| PUPIL    | Eye Tracker pupil diameter                                             |
+| MISC     | Miscellaneous                                                          |
+| SYSCLOCK | System time showing elapsed time since trial started                   |
+| ADC      | Analog to Digital input                                                |
+| DAC      | Digital to Analog output                                               |
+| REF      | Reference channel                                                      |
+| OTHER    | Any other type of channel                                              |
+
```

Comment:
```suggestion
```

I think this empty line needs to be deleted, because our linter doesn't like empty space: https://travis-ci.com/bids-standard/bids-specification/builds/105966063

Reply:
@robertoostenveld if you change this, the CIs would pass - then perhaps we can find somebody else to review this as well (feel free to suggest someone @robertoostenveld ) and then we can merge these changes